docs/HACKING.md

```diff
@@ -1,6 +1,29 @@
 ⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
 
 # Hacking on the WMCO
+Requires all pre-requisites in the [README](/README.md) to be fufilled.
+
+## User Workflows
+
+### I want to install an officially released version of WMCO on my OpenShift/OKD cluster
+* Instructions to install the community operator can be found in the [README](/README.md).
+  Paying customers should instead look at the [OpenShift docs.](https://docs.openshift.com/container-platform/latest/windows_containers/enabling-windows-container-workloads.html)
+
+### I want to try a specific commit of WMCO
```

Comment:
Shouldn't this be `I want to try the unreleased WMCO`? I really don't want to encourage folks trying a specific commit. Folks who are so inclined can do so at their own risk. Applies to all the headers.